test/integration/setup.js

```diff
@@ -138,23 +174,40 @@ const bundleTapeStandalone = async (destDir) => {
 
   const stream = b.bundle();
   const onceStreamEnd = awaitStreamEnd(stream);
-  stream.pipe(fs.createWriteStream(bundleFileName));
+  const destFileStream = fs.createWriteStream(bundleFileName);
+  const onceWritten = new Promise(resolve => {
+    destFileStream.on("close", resolve);
+  });
+  stream.pipe(destFileStream);
 
-  await onceStreamEnd;
+  await Promise.all([onceStreamEnd, onceWritten]);
 };
 
 test.onFailure(() => {
   process.exit(1);
 });
 
-const defineExtensionTests = ({description, extensions}) => {
+const defineExtensionTests = ({description, extensions, exit, skip}) => {
   for (const extensionDirName of extensions) {
     test(`${description} (test extension: ${extensionDirName})`, async (tt) => {
       let timeout;
       let driver;
       let server;
       let tempDir;
 
+      let browser = process.env.TEST_BROWSER_TYPE;
+
+      if (skip) {
+        if (skip === true) {
+          tt.skip("Test extension skipped");
+          return;
+        } else if (skip === browser || skip.includes(browser)) {
```

Comment:
This adds three ways to disable a test: `true`, string, array of strings. Onle the string value is used (`skip: "edge"`). If there is no use for the other two, let's keep it simple for now and only use the string value.

Or remove this alltogether and skip the test in the description inside the test file itself.